tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_1.json

```diff
@@ -51,23 +51,23 @@
                                             "unit": "10^6 cells/mL"
                                         },
                                         "dead cell density (cell counter)": {
-                                            "value": 0.2237332344,
+                                            "value": 0.22373323439999998,
```

Comment:
is this expected? love floating point 🤣 

mostly wondering how did this happen? seems to be happening pretty regularly (when it wasn't before) this is where we divide by `1e6` right? is there a way to preserve sigfigs or something

Reply:
Just for you, I'll make the code less pretty to preserve the values :P